micro_manager/adaptivity/adaptivity.py

```diff
@@ -107,3 +95,49 @@ def _check_for_deactivation(
                 if similarity_dists[active_id, active_id_2] < self._coarse_tol:
                     return True
         return False
+
+    def _get_similarity_measure(self, similarity_measure: str) -> Callable[[np.ndarray], np.ndarray]:
+        """
+        Get similarity measure to be used for similarity calculation
+
+        Parameters
+        ----------
+        similarity_measure : str
+            String specifying the similarity measure to be used
+
+        Returns
+        -------
+        similarity_measure : function
+            Function to be used for similarity calculation. Takes data as input and returns similarity measure
+        """
+        if similarity_measure == 'L1':
+            return self._l1
+        elif similarity_measure == 'L2':
+            return self._l2
+        elif similarity_measure == 'L1rel':
+            return self._l1rel
+        elif similarity_measure == 'L2rel':
+            return self._l2rel
+        else:
+            raise ValueError(
+                'Similarity measure not supported. Currently supported similarity measures are "L1", "L2", "L1rel", "L2rel".')
+
+    def _l1(self, data: np.ndarray) -> np.ndarray:
+        return np.linalg.norm(data[np.newaxis, :] - data[:, np.newaxis], ord=1, axis=-1)
```

Comment:
Can you please add a docstring block to all similarity measure functions, similar to the ones that we have for all other functions?